src/pages/webinars.js

```diff
@@ -0,0 +1,174 @@
+import React from "react";
+import Link from "gatsby-link";
+import withStyles from "elevate-ui/withStyles";
+import WebinarCardGrid from "../components/WebinarCardGrid";
+import WebinarCard from "../components/WebinarCard";
+import Container from "../components/Container";
+import SEO from "../components/SEO";
+import Hexagons from "../images/hexagons.svg";
+
+const Webinars = ({ classes, data }) => {
+  const webinars = data.allContentfulWebinar.edges;
+  return (
+    <div className={classes.root}>
+      <SEO />
+      <Container>
+        <div className={classes.top}>
+          <div className={classes.heading}>Upcoming Webinars</div>
+          <Link className={classes.bootcampLink} to="/bootcamp">
+            Learn about our Social Media Bootcamp →
+          </Link>
+        </div>
+        <WebinarCardGrid className={classes.grid}>
+          {webinars.map(({ node: webinar }) => {
+            // TODO: What's the proper way to do this with static gen?
```

Comment:
Webinars are going to be a little different than events– since we don't have a "date" to order them by– we should find some other way to order them.

I think it would be best to check the day that's come through (that's a string "Monday" "Tuesday" etc., right?) And put them in order accordingly.